integrations/terraform-cloud/main.py

```diff
@@ -165,8 +164,21 @@ async def resync_state_versions(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
         yield enriched_state_versions_batch
 
 
+@ocean.on_resync()
+async def on_create_webhook_resync(kind) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    terraform_client = init_terraform_client()
+    await terraform_client.create_workspace_webhook(
+        app_host=ocean.integration_config["app_host"]
+    )
+    return []
+
+
 @ocean.router.post("/webhook")
 async def handle_webhook_request(data: dict[str, Any]) -> dict[str, Any]:
+    for notifications in data["notifications"]:
+        if notifications["trigger"] == "verification":
+            return {"ok": True}
```

Comment:
Can you share a sample of the webhook verification request data?

Reply:
{'payload_version': 1, 'notification_configuration_id': 'nc-T66N4WGzBovFRhYy', 'run_url': None, 'run_id': None, 'run_message': None, 'run_created_at': None, 'run_created_by': None, 'workspace_id': None, 'workspace_name': None, 'organization_name': None, 'notifications': [{'message': 'Verification of port integration webhook', 'trigger': 'verification', 'run_status': None, 'run_updated_at': None, 'run_updated_by': None}]}